Sources/PrimerSDK/Classes/PCI/Services/API/Primer/PrimerAPI.swift

```diff
@@ -32,14 +32,16 @@ enum PrimerAPI: Endpoint, Equatable {
             (.poll, .poll),
             (.sendAnalyticsEvents, .sendAnalyticsEvents),
             (.createPayment, .createPayment),
-            (.validateClientToken, .validateClientToken):
+            (.validateClientToken, .validateClientToken),
+            (.getNolSdkSecret, .getNolSdkSecret):
             return true
         default:
             return false
         }
     }
     
 
+    case genericEndpoint(clientToken: DecodedJWTToken, url: URL)
```

Comment:
Can we name this e.g. `redirect`? I think this is the only case currently where we'd want to allow the use of URLs that aren't strictly defined as part of the API client. Plus, we use similar terminology for the webview redirect URL that we get from the client token.